percona/templates/pgbouncer-template.json

```diff
@@ -0,0 +1,161 @@
+{
+    "kind": "Deployment",
+    "apiVersion": "apps/v1",
+    "metadata": {
+        "name": "{{.Name}}",
+        "labels": {
+            {{ if .CustomLabels }}
+            {{ .CustomLabels }}
+            {{ end }}
+            "name": "{{.Name}}",
+            "crunchy-pgbouncer": "true",
+            "pg-cluster": "{{.ClusterName}}",
+            "service-name": "{{.Name}}",
+            "vendor": "crunchydata"
+        }
+    },
+    "spec": {
+        "replicas": {{.Replicas}},
+        "selector": {
+            "matchLabels": {
+                "name": "{{.Name}}",
+                "crunchy-pgbouncer": "true",
+                "pg-cluster": "{{.ClusterName}}",
+                "service-name": "{{.Name}}",
+                "{{.PodAntiAffinityLabelName}}": "{{.PodAntiAffinityLabelValue}}",
+                "vendor": "crunchydata"
+            }
+        },
+        "template": {
+            "metadata": {
+                {{ if .PodAnnotations }}
+                "annotations": {{ .PodAnnotations }},
+                {{ end }}
+                "labels": {
+                    {{ if .CustomLabels }}
+                    {{ .CustomLabels }}
+                    {{ end }}
+                    "name": "{{.Name}}",
+                    "crunchy-pgbouncer": "true",
+                    "pg-cluster": "{{.ClusterName}}",
+                    "service-name": "{{.Name}}",
+                    "{{.PodAntiAffinityLabelName}}": "{{.PodAntiAffinityLabelValue}}",
+                    "vendor": "crunchydata"
+                }
+            },
+            "spec": {
+                "serviceAccountName": "pgo-default",
+                "securityContext": {
+                  {{ if not .DisableFSGroup }}
+                  "fsGroup": 2,
+                  {{ end }}
+                  "runAsNonRoot": true
+                },
+                "containers": [{
+                    "name": "pgbouncer",
+                    "image": "{{.Image}}",
+                    "imagePullPolicy": "<imagePullPolicy>",
```

Comment:
Why we can not use` "{{.imagePullPolicy}}" `  as in all other places.

Reply:
because operator will try to find variable `imagePullPolicy` inside struct that it use for fill this template. And using `<` we can understand that this using for our own logic